packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -853,7 +853,12 @@ class ScrollableBase extends Component {
 				for (let i = 0; i < nodes.length; i++) {
 					const nodeBounds = nodes[i].getBoundingClientRect();
 					if (nodeBounds[first] > viewportBounds[first] && nodeBounds[last] < viewportBounds[last]) {
-						Spotlight.focus(nodes[i]);
+						const node = nodes[i];
+						if (node && node.dataset && 'spotlightContainer' in node.dataset) {
```

Comment:
Is there reason to use 'spotlightContainer' in node.dataset instead of node.dataset.spotlightContainer?

Reply:
I think we can use node.dataset.spotlightContainer, but just want to be consistence with isContainerNode() in container.js